packages/itinerary-body/src/styled.js

```diff
@@ -417,6 +417,7 @@ export const PlaceName = styled.div`
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
+  flex: 1 1 auto;
```

Comment:
We need to agree where the map button needs to be. @landonreed would you like to provide your opinion?

![image](https://user-images.githubusercontent.com/56846598/98981177-a2ab9380-24eb-11eb-8a79-d35a671b51f8.png)